### PR TITLE
Update badges in README.md

### PR DESCRIPTION
**Pull request by bastelfreak**
some links were outdated or simply wrong.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+[![Build Status](https://github.com/voxpupuli/puppet-allknowingdns/workflows/CI/badge.svg)](https://github.com/voxpupuli/puppet-allknowingdns/actions?query=workflow%3ACI)
+[![Release](https://github.com/voxpupuli/puppet-allknowingdns/actions/workflows/release.yml/badge.svg)](https://github.com/voxpupuli/puppet-allknowingdns/actions/workflows/release.yml)
+[![Puppet Forge](https://img.shields.io/puppetforge/v/puppet/allknowingdns.svg)](https://forge.puppetlabs.com/puppet/allknowingdns)
 [![Puppet Forge - downloads](https://img.shields.io/puppetforge/dt/puppet/allknowingdns.svg)](https://forge.puppetlabs.com/puppet/allknowingdns)
 [![Puppet Forge - endorsement](https://img.shields.io/puppetforge/e/puppet/allknowingdns.svg)](https://forge.puppetlabs.com/puppet/allknowingdns)
 [![Puppet Forge - scores](https://img.shields.io/puppetforge/f/puppet/allknowingdns.svg)](https://forge.puppetlabs.com/puppet/allknowingdns)
-[![License](http://img.shields.io/:license-gpl3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0.html)
+[![puppetmodule.info docs](http://www.puppetmodule.info/images/badge.png)](http://www.puppetmodule.info/m/puppet-allknowingdns)
```
Comment (smortex) [inline README.md:9]: What is this puppetmodule.info?  Does not resolve to a valid address here… :thinking: 

```sh-session
romain@zappy ~ % host www.puppetmodule.info               
www.puppetmodule.info is an alias for www.puppetmodule.info.herokudns.com.
```

**Review by dhollinger — APPROVED** [review]

**Review by smortex — COMMENTED** [review]
Not sure about puppetmodule.info (is it currently having trouble?), the rest looks good.

File: README.md (hunk 1 above)
Comment (smortex) [reply README.md:9]: Cannot run a worldwide test…

RIPE Atlas screenshot saying I can't create a measurement to check this worldwide

Comment (bastelfreak) [reply README.md:9]: it's currently having troubles and will hopefully come back tomorrow, from a new owner/server! (spoiler: https://github.com/voxpupuli/puppetmodule.info)

Comment (smortex) [reply README.md:9]: I **knew** I saw something matching that name recently…

**Pull request merged**